### ISSUE TITLE
Contrasto per menu' in h3 da rivedere (pagina-template)

### ISSUE THREAD

**Issue by rzaupa** [Gruppo-TecWeb/Codice#130]
There is insufficient color contrast between the foreground and background of large text content.
WCAG 2.1 Success Criterion:AA 1.4.3 Contrast (Minimum) `<h3 id="menu-title"></h3>`

https://www.w3.org/TR/WCAG21/#contrast-minimum
